taxonomies/ingredients.txt

```diff
@@ -27,6 +27,18 @@ synonyms:es:jarabe, sirope
 
 synonyms:fr:semi-complet, demi-complet
 
+synonyms:es:semi-completo, medio completo
+
+synonyms:de:halb-vollständig, halb-komplett
+
+synonyms:it:semi-completo, mezzo completo
```

Comment:
In italian it's "integrale", I don't think it's "completo".

And I dont think mezzo integrale exists…